staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -152,32 +152,74 @@ Get-CimInstance win32_pagefileusage | Format-List * | Out-File -Append "$ENV:TEM
 Get-CimInstance win32_computersystem | Format-List AutomaticManagedPagefile | Out-File -Append "$ENV:TEMP\\$($timeStamp)_pagefile.txt"
 $paths += "$ENV:TEMP\\$($timeStamp)_pagefile.txt"
 
+Write-Host "Get disk usage for all drives"
```

Comment:
Suggest to adding try/catch blocks for below steps to avoid that any error will block users to collect other logs.
1. Get-PSDrive
2. cri.exe
3. du.exe
4. crictl.exe
5. containerd.exe

Reply:
line 2 has `$ProgressPreference = "SilentlyContinue"` can continue to run

Reply:
Tested. No need of `try catch`

![image](https://github.com/Azure/AgentBaker/assets/62196586/739c1000-ffb5-4a95-b9b7-f080d94582ed)